internal/nix/install.go

```diff
@@ -154,31 +75,35 @@ func EnsureNixInstalled(writer io.Writer, withDaemonFunc func() *bool) (err erro
 
 	color.Yellow("\nNix is not installed. Devbox will attempt to install it.\n\n")
 
+	installer := nix.Installer{}
 	if isatty.IsTerminal(os.Stdout.Fd()) {
 		color.Yellow("Press enter to continue or ctrl-c to exit.\n")
 		fmt.Scanln() //nolint:errcheck
-	}
 
-	if err = Install(writer, withDaemonFunc); err != nil {
-		return err
-	}
+		spinny := spinner.New(spinner.CharSets[11], 100*time.Millisecond, spinner.WithWriter(writer))
+		spinny.Suffix = " Downloading the Nix installer..."
```

Comment:
nit: drop the prefix space?

Reply:
I think this puts a space between the spinner and the message. I copied what we have for [`Computing the Devbox environment`](https://github.com/jetify-com/devbox/blob/d2957086a837cd7fe93ce8fe8556e446835d5f0a/internal/devbox/devbox.go#L592).